hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -202,11 +209,11 @@ private OzoneManagerDoubleBuffer(OMMetadataManager omMetadataManager,
         OzoneManagerDoubleBufferMetrics.create();
     this.indexToTerm = indexToTerm;
     this.flushNotifier = flushNotifier;
-
+    this.threadPrefix = (threadPrefix == null) ? "" : threadPrefix;
```

Comment:
We can avoid `null` prefix by setting `""` as initial value in the builder:

https://github.com/apache/ozone/blob/02d3f30030f0b95e430de366d6be8af9022d68bf/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java#L128

Reply:
Thanks will implement the change!